topics/babel/index.md

```diff
@@ -0,0 +1,12 @@
+---
+created_by: Sebastian McKenzie, James Kyle, Henry Zhu, Logan Smyth, Daniel Tschinder
+display_name: Babel
+github_url: https://github.com/babel
+logo: babel.png
+released: Sep 28, 2014
+short_description: 🐠 Babel is a compiler for writing next generation JavaScript, today.
+topic: babel
+url: https://babeljs.io/
+---
+
+🐠 Babel is a compiler created by [Sebastian McKenzie](https://github.com/kittens) in 2014 originally created to convert ES6 to ES5 (previously 6to5), but now is a toolchain to allow developers to write any next generation JavaScript and testing ground for proposals from [TC39](https://github.com/tc39), the technical committee that specifies [ECMAScript](https://en.wikipedia.org/wiki/ECMAScript). Babel can also convert [JSX syntax](https://facebook.github.io/jsx/) and strip out type annotations from both [Flow](https://flow.org/) and [TypeScript](https://www.typescriptlang.org/). Babel is built out of plugins. Compose your own transformation pipeline using [plugins written by the community](https://www.npmjs.com/browse/keyword/babel-plugin) or [write your own](https://github.com/thejameskyle/babel-handbook).
```

Comment:
first sentence has `created` twice

Reply:
I suggested to @hzoo:

> Babel is a compiler created by Sebastian McKenzie in 2014 to convert ES6 to ES5 (originally called 6to5). It has since become a toolchain that enables developers to write any next generation JavaScript and serves as a testing ground for proposals from TC39, the technical committee that specifies ECMAScript.